src/Pages/MyPage/MyPageComponent/LeftMenuCard.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
+import { IoIosArrowForward } from "react-icons/io";
+
+const LeftMenuCard = (props) => {
+  return (
+    <LeftMenuCardContainer bold={props.menuTitle}>
+      <div className="cardTitle">{props.menuTitle}</div>
+      <div className="cardMenu">
+        {props.menuList &&
+          props.menuList.map((list, idx) => {
```

Comment:
```suggestion
        {menuList?.map((list, idx) => {
```
- 함수 파라미터 자리에서 구조분해 할당 가능합니다
- 옵셔널 체이닝 적용 가능합니다

Reply:
수정했습니다!